dashboard-frontend/src/pages/ConnectPage/ConnectAccountsList.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { AccountResource } from '../../types/account';
+import { classNames } from '../../helpers/strings';
+
+import { Link } from 'react-router-dom';
+
+const ConnectedAcccountsList = ({
+  accounts,
+}: {
+  accounts: AccountResource[];
+}) => {
+  return (
+    <ListLayout>
+      {accounts.map((account, index) => (
+        <AccountListItem key={account.id} account={account} />
+      ))}
+    </ListLayout>
+  );
+};
+
+const AccountListItem = ({
+  account: { name, slug, authType },
+}: {
+  account: AccountResource;
+}) => (
+  <li>
+    <div className="px-4 py-4 sm:px-6">
+      <div className="flex flex-col sm:flex-row sm:items-center">
+        <p className="text-sm font-medium text-indigo-600 truncate">{name}</p>
+
+        {authType === 'secret' ? (
```

Comment:
It'd be a lot easier to read if these were just separate (if similar) components!